ratelimit/ratelimit.go

```diff
@@ -292,7 +292,7 @@ func (voidRatelimit) RetryAfter(string) int      { return 0 }
 func (voidRatelimit) Delta(string) time.Duration { return -1 * time.Second }
 func (voidRatelimit) Resize(string, int)         {}
 
-func newRatelimit(s Settings, sw Swarmer) *Ratelimit {
+func newRatelimit(s Settings, sw Swarmer, so *swarm.Options, redisRing *ring) *Ratelimit {
```

Comment:
see the comment in newClusterRateLimiter: we can drop the so *swarm.Options parameter here, too, and keep only the sw Swarmer parameter.